documentation/user_docs/docs/manual/Manipulating_and_extracting_data_from_SQW_files_and_objects.rst

```diff
@@ -174,46 +375,62 @@ save_xye
 
-::
+.. code-block:: matlab
 
-   filename='C:\\mprogs\\my_ascii_file.txt';
+   filename = 'C:\\mprogs\\my_ascii_file.txt';
    save_xye(w_in,filename);
 
 
-The format of the ascii file for an n-dimensional dataset is n columns of co-ordinates along each of the axes, plus one column of signal and another column of error (standard deviation).
+The format of the ascii file for an n-dimensional dataset is n columns of co-ordinates along each of the axes, plus one
+column of signal and another column of error (standard deviation).
 
 
 xye
 ===
 
 Extract the bin centres, intensity and standard errors from an sqw or dnd object.
 
-::
+.. code-block:: matlab
 
-   S=xye(w);
+   S = xye(w);
 
 
-The output is a structure with fields S.x (bin centres if a 1D object, or cell array of vectors containing the bin centres along each axis if 2D, 3D or 4D object), S.y (array of intensities), S.e (array of estimated error on the intensities).
+The output is a structure with fields S.x (bin centres if a 1D object, or cell array of vectors containing the bin
+centres along each axis if 2D, 3D or 4D object), S.y (array of intensities), S.e (array of estimated error on the
+intensities).
 
 
 hkle
 ====
 
-Obtain the reciprocal space coordinate [h,k,l,e] for points in the coordinates of the display axes for an sqw object **from a single spe file**
+Obtain the reciprocal space coordinate [h,k,l,e] for points in the coordinates of the display axes for an sqw object
+**from a single spe file**
 
-::
+.. code-block:: matlab
 
     [qe1,qe2] = hkle(w,x)
 
 
 The inputs take the form:
 
-``w`` - sqw object
-``x`` - Vector of coordinates in the display axes of an sqw object. The number of coordinates must match the dimensionality of the object. e.g. for a 2D sqw object, ``x=[x1,x2]``, where ``x1``, ``x2`` are column vectors. More than one point can be provided by giving more rows e.g. ``[1.2,4.3; 1.1,5.4; 1.32, 6.7]`` for 3 points from a 2D object. Generally, an (n x nd) array, where n is the number of points, and nd the dimensionality of the object.
+* ``w``
+
+  sqw object
+
+* ``x``
+
+  Vector of coordinates in the display axes of an sqw object. The number of coordinates must match the dimensionality of
+  the object. e.g. for a 2D sqw object, ``x = [x1,x2]``, where ``x1``, ``x2`` are column vectors. More than one point can
+  be provided by giving more rows e.g. ``[1.2,4.3; 1.1,5.4; 1.32, 6.7]`` for 3 points from a 2D object. Generally, an
+  (``n`` x ``nd``) array, where ``n`` is the number of points, and ``nd`` the dimensionality of the object.
 
 The outputs take the form:
 
 
-``qe1`` - Components of momentum (in rlu) and energy for each bin in the dataset. Generally, will be (n x 4) array, where n is the number of points
+* ``qe1``
+
+  Components of momentum (in rlu) and energy for each bin in the dataset. Generally, will be (n x 4) array, where n is the number of points
+
+* ``qe2``
 
-``qe2`` - For the second root
+  For the second root
```

Comment:
incomplete

Reply:
I'm not sure it is... What it means, I don't know. This is what was there originally